app/modules/onboarding/module.php

```diff
@@ -440,6 +439,10 @@ private function maybe_handle_ajax() {
 				break;
 			case 'elementor_update_onboarding_option':
 				$result = $this->maybe_update_onboarding_db_option();
+				break;
+			case 'elementor_save_onboarding_features':
+				// phpcs:ignore WordPress.Security.NonceVerification.Missing
```

Comment:
How not heaving a nonce can impact here?
Can it expose this function to external requests?

Reply:
Nonce is checked before calling `maybe_handle_ajax`. Here:  
https://github.com/elementor/elementor/pull/28193/files#diff-ba434f169932cab25396a4f9ddbdc58723a0eb014ab202d6287882ffccfa5783R485